shark/examples/shark_inference/stable_diffusion/opt_params.py

```diff
@@ -23,13 +23,8 @@ def get_unet():
     # Tuned model is present for `fp16` precision.
     if args.precision == "fp16":
         if args.use_tuned:
-            sys.exit("The tuned version is in maintenance, stay tuned.")
             bucket = "gs://shark_tank/quinn"
-            model_name = "unet_fp16_tunedv2"
-            iree_flags += [
-                "--iree-flow-enable-padding-linalg-ops",
-                "--iree-flow-linalg-ops-padding-size=32",
-            ]
+            model_name = "unet_22nov_fp16_tuned"
             # TODO: Pass iree_flags to the exported model.
             if args.import_mlir:
```

Comment:
I think this check can be removed here unless we are importing and adding the tuned config on the fly.

Reply:
Makes sense, I removed it.